examples/README.md

```diff
@@ -146,3 +146,14 @@ The vigor example shows how to integrate [ASP.NET Core health checks](https://do
 * Hosting gRPC Health Checking Protocol service
 * Integrate [ASP.NET Core health checks](https://docs.microsoft.com/aspnet/core/host-and-deploy/health-checks) with gRPC health checks
 * Calling service with `Grpc.HealthCheck` client
+
+## [Browser](./Browser)
+
+The browser example shows how to use [gRPC-Web](https://github.com/grpc/grpc-web) with ASP.NET Core to call a gRPC service from a browser. This example requires [npm and NodeJS](https://nodejs.org/) to be installed on your computer.
```

Comment:
just curious: are you going to add an example for .NET grpc-web client using Blazor/WebAssembly as well? It's an interesting piece of technology in that it allows one to use the same gRPC C# client API regardless whether you're running in the browser or if you're a microservice in a datacenter.

Reply:
I will add it as a separate sample.